multi-server-adding-mailbox.adoc

```diff
@@ -0,0 +1,228 @@
+[[multi-server-adding-mailbox]]
+= Adding a Mailbox Server to a Single Server Configuration
+
+In the {product-name} ({product-abbrev}) single server environment,
+the LDAP, MTA, and mailbox services are on one machine. This chapter
+explains how to add a new machine that is configured as a mailbox
+server to a single server configuration and how to remove the mailbox
+server from the single server node.
+
+== Setup Requirements For Adding a Mailbox Server
+
+* The new machine you are adding must have the same operating system,
```

Comment:
Greg should that part be yanked?  Are we comfortable with removing that restriction from our published docs?